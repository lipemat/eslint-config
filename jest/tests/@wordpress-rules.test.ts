import jestRunnerEslint from '../helpers/jest-runner-eslint';

describe( 'WordPress rules are enabled', () => {
	test( '@wordpress/invalid-text-domain failing', async () => {
		expect( await jestRunnerEslint( 'failing/@wordpress/i18n-text-domain' ) ).toMatchSnapshot();

		expect( await jestRunnerEslint( 'passing/@wordpress/i18n-text-domain' ) ).toMatchSnapshot();
	} );
} );
