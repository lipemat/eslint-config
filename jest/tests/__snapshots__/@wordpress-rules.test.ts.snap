// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WordPress rules are enabled @wordpress/invalid-text-domain failing 1`] = `
"FAIL jest/fixtures/failing/@wordpress/i18n-text-domain/invalid-text-domain.ts
  × @wordpress/i18n-text-domain (mocked ms)
  × @wordpress/i18n-text-domain (mocked ms)


E:\\SVN\\the-boilerplate\\packages\\eslint-config\\jest\\fixtures\\failing\\@wordpress\\i18n-text-domain\\invalid-text-domain.ts
  3:15  error  Missing text domain          @wordpress/i18n-text-domain
  7:9   error  Invalid text domain 'wrong'  @wordpress/i18n-text-domain

✖ 2 problems (2 errors, 0 warnings)
  2 errors and 0 warnings potentially fixable with the \`--fix\` option.

Test Suites: 1 failed, 1 total
Tests:       2 failed, 2 total
Snapshots:   0 total
Time:        
Ran all test suites.
"
`;

exports[`WordPress rules are enabled @wordpress/invalid-text-domain failing 2`] = `
"PASS jest/fixtures/passing/@wordpress/i18n-text-domain/correct-usage.ts
  √ ESLint (mocked ms)

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        
Ran all test suites.
"
`;
